soda/scientific/soda/scientific/distribution/generate_dro.py

```diff
@@ -126,13 +125,10 @@ def generate_continuous_dro(self) -> DRO:
         return DRO(weights=weights.tolist(), bins=bins.tolist())
 
     def generate_categorical_dro(self) -> DRO:
-        data = pd.Series(self.data)
-        value_counts = data.value_counts()
-
-        labels = value_counts.index.to_numpy()
-        weights = value_counts.to_numpy()
-        weights = normalize(weights)
-        return DRO(weights=weights.tolist(), bins=labels.tolist())
+        labels = [row[0] for row in self.data]
+        label_counts = [row[1] for row in self.data]
+        weights = [label_count / sum(label_counts) for label_count in label_counts]
```

Comment:
what if `sum(label_counts)=0`? Can it be the case?

Reply:
I believe they can never be zero ever or else this line would have thrown an error: [ if not column_values](https://github.com/sodadata/soda-core/pull/1847/files#diff-bb85d418b19f0af8eb3a93cbf5963e15aeaabb3bced924989cece5888979c7e4R338-R342)